examples/aditof-demo/aditofdemoview.cpp

```diff
@@ -1100,7 +1112,7 @@ void AdiTofDemoView::_displayIrImage() {
 
         m_irImage = cv::Mat(frameHeight, frameWidth, CV_16UC1, irData);
         m_irImage.convertTo(m_irImage, CV_8U, 255.0 / max_value_of_IR_pixel);
-        flip(m_irImage, m_irImage, 1);
+        flip(m_irImage, m_irImage, 0);
```

Comment:
Should flip on the horizontal axis only if SMART_3D is defined
Do not flit on the vertical axis anymore